packages/ui-components/src/components/date-time-input/date-time-input.tsx

```diff
@@ -137,7 +142,7 @@ export class DateTimeInput implements IDateTimeInput, IDateTimeInputEvents {
 		return (
 			<Host>
 				<div class="date-time-input-container">
-					<kv-form-label label={this.label} />
+					<kv-form-label label={this.label} required={this.required} />
```

Comment:
Can it be required if it's disabled?

Reply:
I assumed that this can be required even though is disabled because imagine a use case where the disable is only temporary and its changed after the user does a specific action. Even though he cannot update this input, it continues to be a required field.

Reply:
Fine by me then 👍